### PR TITLE
Fixed some small issues with project setup

### PR DESCRIPTION
**Pull request by MEseifan**

### CODE REVIEW

File: src/main/java/io/cdap/plugin/dlp/SensitiveRecordFilter.java [hunk 1]
```diff
@@ -103,8 +102,8 @@ public void configurePipeline(MultiOutputPipelineConfigurer configurer) {
 
       client = DlpServiceClient.create(getSettings());
     } catch (IOException e) {
-      LOG.error("Failed to create DLP Client: "+e.printStackTrace());
-      stageConfigurer.getFailureCollector().addFailure("Failed to create DLP client: "+e.getMessage());
+      LOG.error("Failed to create DLP Client: " + e.getMessage());
```
Comment (CuriousVini) [inline src/main/java/io/cdap/plugin/dlp/SensitiveRecordFilter.java]: log.error("message: {}", e.getmessage(), e)

File: src/main/java/io/cdap/plugin/dlp/SensitiveRecordRedaction.java [hunk 2]
```diff
@@ -313,7 +313,7 @@ public void validate(FailureCollector collector, Schema inputSchema) {
       if (fieldsToTransform != null) {
         try {
           List<DlpFieldTransformationConfig> transformationConfigs = parseTransformations();
-          HashMap<string, string> transforms = new HashMap<>();
+          HashMap<String, String> transforms = new HashMap<>();
```
Comment (CuriousVini) [inline src/main/java/io/cdap/plugin/dlp/SensitiveRecordRedaction.java:316]: Map<String,String>

File: src/main/java/io/cdap/plugin/dlp/Utils.java [hunk 3]
```diff
@@ -32,6 +32,7 @@
 import java.time.ZoneId;
 import java.time.ZonedDateTime;
 import java.time.format.DateTimeFormatter;
+import java.util.stream.Collectors;
```
Comment (CuriousVini) [inline src/main/java/io/cdap/plugin/dlp/Utils.java:35]: why is this needed?

File: src/main/java/io/cdap/plugin/dlp/configs/DlpFieldTransformationConfigCodec.java [hunk 4]
```diff
@@ -21,11 +21,10 @@
 import com.google.gson.JsonElement;
 import com.google.gson.JsonObject;
 import com.google.gson.JsonParseException;
-import com.google.gson.JsonSerializationContext;
-import com.google.gson.JsonSerializer;
 
 import java.lang.reflect.Type;
 import java.util.HashMap;
+import java.util.Map;
```
Comment (CuriousVini) [inline src/main/java/io/cdap/plugin/dlp/configs/DlpFieldTransformationConfigCodec.java:27]: why is this needed?

File: src/main/java/io/cdap/plugin/dlp/Utils.java (hunk 3 above)
Comment (MEseifan) [reply src/main/java/io/cdap/plugin/dlp/Utils.java:35]: We call `Collectors.toList()` on line 57

File: src/main/java/io/cdap/plugin/dlp/configs/DlpFieldTransformationConfigCodec.java (hunk 4 above)
Comment (MEseifan) [reply src/main/java/io/cdap/plugin/dlp/configs/DlpFieldTransformationConfigCodec.java:27]: We use a Map on line 34

**Review by CuriousVini — APPROVED** [review]
